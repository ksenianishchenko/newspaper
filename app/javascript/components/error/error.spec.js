import React from "react";
import renderer from 'react-test-renderer';
import Error from "./error";


describe("Error message", () => {

  it('should render correctly error message', () => {
      const tree = renderer.create(<Error>This is error message</Error>).toJSON();
      expect(tree).toMatchSnapshot();
  });

})
