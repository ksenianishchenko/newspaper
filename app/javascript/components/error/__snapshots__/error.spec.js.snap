// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message should render correctly error message 1`] = `
<p
  className="text-danger"
>
  This is error message
</p>
`;
